Dockerfile

```diff
@@ -0,0 +1,62 @@
+# https://github.com/letsencrypt/lets-encrypt-preview/pull/431#issuecomment-103659297
+# it is more likely developers will already have ubuntu:trusty rather
+# than e.g. debian:jessie and image size differences are negligible
+FROM ubuntu:trusty
+MAINTAINER Jakub Warmuz <jakub@warmuz.org>
+
+# You neccesarily have to bind to 443@host as well! (ACME spec)
+EXPOSE 443
+
+# TODO: make sure --config-dir and --work-dir cannot be changed
+# through the CLI (letsencrypt-docker wrapper that uses standalone
+# authenticator and text mode only?)
+VOLUME /etc/letsencrypt /var/lib/letsencrypt
+
+WORKDIR /opt/letsencrypt
+
+# no need to mkdir anything:
+# https://docs.docker.com/reference/builder/#copy
+# If <dest> doesn't exist, it is created along with all missing
+# directories in its path.
+
+# The following copies too much than we need...
+#COPY . /opt/letsencrypt/
+
+COPY bootstrap/debian.sh /opt/letsencrypt/src/
```

Comment:
This should probably say ubuntu.sh now that it's based on an Ubuntu image, in case the two bootstrap scripts diverge in the future.